Community-Aid/ca-frontend
# Auth flow

Support these flows:
1. Create account -> verify account -> home -> profile -> logout
2. Create account -> unable to verify -> logout -> create account
3. Login -> home -> profile -> logout

Not yet supported:
1. Create account -> unable to verify -> resend email
2. Login -> forgot password

https://user-images.githubusercontent.com/4309735/143394037-7d52f06e-8d40-4c93-b323-1732cc9baf20.mov
